### ISSUE TITLE
[Bug]: Unable to disable 'Accessibility' theme in NC 25

### ISSUE THREAD

**Issue by LeoWare** [nextcloud/server#38744]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

Hi. The 'Accessibility' app has been added to the core. The new theme breaks the theme I want (Breeze Dark). I self-host. I am the only user. I do not use a screen reader. Forcing this on self-hosted installs is not ok. Allow me to administer my server how I want.

### Steps to reproduce

1. Install Nextcloud 25
2. See accessibility theme has been turned on without the administrator's permission.
3. See no way to disable accessibility theme.

### Expected behavior

Accessibility theme needs a disable button.

### Installation method

Community Manual installation with Archive

### Nextcloud Server version

25

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 7.4

### Web server

Apache (supported)

### Database engine version

MySQL

### Is this bug present after an update or on a fresh install?

Upgraded to a MAJOR version (ex. 22 to 23)

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "206.81.2.156",
            "cloud.samuelraynor.com"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "25.0.7.1",
        "overwrite.cli.url": "http:\/\/206.81.2.156",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "mysql.utf8mb4": true,
        "maintenance": false,
        "theme": "",
        "loglevel": 0,
        "app_install_overwrite": [
            "twofactor_yubikey"
        ],
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_sendmailmode": "smtp",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpsecure": "tls",
        "mail_smtpauth": 1,
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "587",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauthtype": "PLAIN",
        "twofactor_enforced": "false",
        "twofactor_enforced_groups": [],
        "twofactor_enforced_excluded_groups": [],
        "memcache.local": "\\OC\\Memcache\\APCu",
        "default_phone_region": "US",
        "updater.secret": "***REMOVED SENSITIVE VALUE***",
        "enforce_theme": "breezedark"
    }
}
```

**Comment by szaimen** [comment]
Thank you for taking the time to report a bug! 👍

As this seems to be a setup issue I would like to ask you to raise your question in the forums: https://help.nextcloud.com

**Comment by LeoWare** [comment]
Why woupd I ask in the forums? If there is an option to disable this, then you should be able to tell me here. If it doesn't exist, no amount of asking in the forum is going to help.I'll have to assume that you're not willing to fix this obvious oversight.Thanks for ruining my installation.Have a wonderful day.
-------- Original message --------From: "Simon L." ***@***.***> Date: 6/11/23  11:06 PM  (GMT-08:00) To: nextcloud/server ***@***.***> Cc: Samuel Raynor ***@***.***>, Author ***@***.***> Subject: Re: [nextcloud/server] [Bug]: Unable to disable 'Accessibility' theme in NC 25 (Issue #38744) 
Thank you for taking the time to report a bug! 👍
As this seems to be a setup issue I would like to ask you to raise your question in the forums: https://help.nextcloud.com

**Comment by szaimen** [comment]
> Why woupd I ask in the forums? If there is an option to disable this

This is a bug tracker and not meant for questions. Ans there is indeed a way to disable this.

**Comment by LeoWare** [comment]
Then you should have linked to the documentation that says this.Closing this saying 'go to the forum ' is lazy and hostile to users.Not having a visible switch for this in the UI is, indeed, a bug.Have a wonderful day. No need to respond any longer. You're ignoring a huge bug. Good bye.
-------- Original message --------From: "Simon L." ***@***.***> Date: 6/11/23  11:26 PM  (GMT-08:00) To: nextcloud/server ***@***.***> Cc: Samuel Raynor ***@***.***>, Author ***@***.***> Subject: Re: [nextcloud/server] [Bug]: Unable to disable 'Accessibility' theme in NC 25 (Issue #38744) 

Why woupd I ask in the forums? If there is an option to disable this

This is a bug tracker and not meant for questions. Ans there is indeed a way to disable this.

**Comment by szaimen** [comment]
I will just repeat myself:

> This is a bug tracker and not meant for questions. Ans there is indeed a way to disable this.
